src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5033,7 +5033,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>Invalid name for a preprocessing symbol; '{0}' is not a valid identifier</value>
   </data>
   <data name="ERR_FeatureNotAvailableInVersion7_1" xml:space="preserve">
-    <value>Feature '{0}' is not available in C# 7.1.  Please use language version {1} or greater.</value>
+    <value>Feature '{0}' is not available in C# 7.1. Please use language version {1} or greater.</value>
+  </data>
+  <data name="ERR_LanguageVersionCannotHaveLeadingZeroes" xml:space="preserve">
+    <value>Specified language version '{0}' cannot have leading zeroes</value>
```

Comment:
This isn't about a breaking change, it's about symmetry.  

``` txt
/langversion:07
Error: leading zeros not allowed 
/langversion:7.00
Invalid language version 
```

Seems like this could be a reasonable typo in the new point release world.  

Not hard feedback.  Just an item I want to make sure we've considered.